plugins/TelescopeControl/CMakeLists.txt

```diff
@@ -2,8 +2,8 @@
 # WARNING: Update also the version number in resources/device_models.json,
 # otherwise the local copy will be overwritten each time Stellarium is started.
 SET(TELESCOPE_CONTROL_MAJOR "0")
-SET(TELESCOPE_CONTROL_MINOR "3")
-SET(TELESCOPE_CONTROL_PATCH "1")
+SET(TELESCOPE_CONTROL_MINOR "4")
```

Comment:
@gzotti Is that something I should be doing? I though that this would be a good time to bump the minor version

Reply:
Yes, seems reasonable to do. Please also follow the Warning to update resources/device_models.json. But @alex-w knows the plugin much better than I do.

Reply:
@alex-w can you elaborate on the device_models.json file? Looking at it, some of those devices also support ASCOM and then there's a huge list of devices which will additionally be supported by the direct support of ASCOM now. How should we handle this? Where is this file used?

Reply:
Just update the version in device_models.json file - he is used for define supported devices for our own drivers of mounts.